spec/certs/store_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe 'Certificate store in config/certs' do
+  before do
+    # We need to allow net connect to download CRLs and check for revocations
+    WebMock.allow_net_connect!
```

Comment:
Would it be worth the effort to whitelist domains for this?

```
Webmock.disable_net_connect!(allow: [
  'ocsp.lol.gov',
  'certs.mil'
])
```

Reply:
I thought there'd be a bunch but after digging in there are only 4. I added them to an allow list.